source/docs/use/download/download.md

```diff
@@ -2,11 +2,11 @@
 title: Downloading HACS
 description: "Step-by-step instructions on downloading HACS to your Home Assistant"
 ---
-Setting up HACS is done by using the [HACS download script](https://github.com/hacs/get).
+To set up HACS, you can use the [HACS download script](https://github.com/hacs/get).
```

Comment:
As you will not use this directly for OS/supervised installations, let's move this to the core/container parts.